src/configureApplication.tsx

```diff
@@ -1,8 +1,12 @@
 import log from 'loglevel';
 import Raven from 'raven-js';
-import { library } from '@fortawesome/fontawesome-svg-core';
+import { config, library } from '@fortawesome/fontawesome-svg-core';
 import { fas } from '@fortawesome/free-solid-svg-icons';
 import { far } from '@fortawesome/free-regular-svg-icons';
+import '@fortawesome/fontawesome-svg-core/styles.css';
+
+// Turn off inline CSS injection for CSP.
+config.autoAddCss = false;
```

Comment:
To keep the logic of this module consistent, this should move inside the body of `configureApplication()`.

Reply:
The docs say this needs to be called before any other fontawesome APIs. If this is moved into a function its it guaranteed to be run early enough both now and in the future?

Reply:
> If this is moved into a function is it guaranteed to be run early enough both now and in the future?

The `configureApplication()` function already contains some FontAwesome code to initialize the lib, so it'd say yes.